node/core/pvf/src/execute/queue.rs

```diff
@@ -225,9 +225,9 @@ fn handle_job_finish(
 	result_tx: ResultSender,
 ) {
 	let (idle_worker, result) = match outcome {
-		Outcome::Ok { result_descriptor, duration_ms, idle_worker } => {
+		Outcome::Ok { result_descriptor, duration_ms: _, idle_worker } => {
 			// TODO: propagate the soft timeout
-			drop(duration_ms);
+			//drop(duration_ms);
```

Comment:
?

Reply:
I was scared to remove this as I don't know what the intention here was with manually dropping it. Can it go?

Reply:
It can go. There's no reason for the `drop`, it's just a `u64`. I removed it in https://github.com/paritytech/polkadot/pull/6282 and just renamed the variable to `_duration_ms`.